server.js

```diff
@@ -132,6 +132,7 @@ app.post('/getParticularSuperlative', (req, res, next) => {
 // =======================
 
 app.post('/addEvent', (req, res, next) => {
+  console.log('addEvent', req.body);
```

Comment:
Handling this now would be more trouble than it's worth, but let's make sure to nix this line in the future (when we do the code review refactoring/polishing at the end)

Reply:
Oops. Actually, let me try taking it out now. I'm pretty sure it's painless. I just haven't updated a PR in a while.